odh-dashboard/base/model-serving-role.yaml

```diff
@@ -5,9 +5,15 @@ metadata:
 rules:
   - verbs:
       - get
+      - list
     apiGroups:
-      - ''
+      - template.openshift.io
     resources:
-      - configmaps
-    resourceNames:
-      - servingruntimes-config
+      - templates
```

Comment:
We have no references to fetching the old secret, right? 🤔

Reply:
Nope, I've wipped that from the code and change the logic in the deployer